lib/pure/json.nim

```diff
@@ -363,10 +392,25 @@ proc `[]=`*(obj: JsonNode, key: string, val: JsonNode) {.inline.} =
   assert(obj.kind == JObject)
   obj.fields[key] = val
 
+template noserialize*() {.pragma.}
```

Comment:
If possible please add documentation to this as well.

Reply:
Will do!